### ISSUE TITLE
[15.0] web_responsive: Use search more... dialog in responsive views on m2x widget

### ISSUE THREAD

**Issue by anhjean** [OCA/web#2214]
I use Odoo 15 CE and web_resposive module

I hope the fields in list view can show full length of the field text content

![Screenshot_2022-05-10-23-39-24-66_e4424258c8b8649f6e67d283a50a2cbc](https://user-images.githubusercontent.com/51810762/167679573-383032fb-5f3a-4d29-b3a3-b11ea994d0ce.jpg)

**Comment by pedrobaeza** [comment]
No, that's not possible. For mobile, kanban views should be used instead, or have such strip.

**Comment by anhjean** [comment]
Can we change to kanban view for the product page that be shown when we add/change the product of an order.?

Link demo clip for issue above:' https://drive.google.com/file/d/1M1arN9cDJq_2Qh9KTs0nAuuaO7kXB88h/view?usp=drivesdk

**Comment by pedrobaeza** [comment]
OK, I see the problem.

@SplashS why putting the "Search more..." dialog when searching in a many2one instead of the usual list? Or is something of the core?

**Comment by SplashS** [comment]
> why putting the "Search more..." dialog when searching in a many2one instead of the usual list? Or is something of the core?

When I was working on mobile UI for form I've been found that dropdown list is hard to be customized. But it have problems on mobile such as dropping down out of the screen, may be more. And I used solution from Enterprise. They show full screen search for list in mobile. I ported this solution.

Indeed in Enterprise this dialog is showed in kanban mode. I don't remember any idea why I left list view.
May be it's happened by mistake. I can change default view for this dialog to kanban.
It will take time. I think about a week.
